tools/report.sh

```diff
@@ -319,6 +319,7 @@ mapfile -t KAFKA_PODS < <($KUBE_CLIENT -n "$NAMESPACE" get po -l strimzi.io/kind
 if [[ ${#KAFKA_PODS[@]} -ne 0 ]]; then
   for pod in "${KAFKA_PODS[@]}"; do
     echo "    $pod"
+    $KUBE_CLIENT exec -i "$pod" -n "$NAMESPACE" -c kafka -- kill -SIGQUIT 1
```

Comment:
I agree the kafka pod should be the main target for the thread dump, but I think all other java processes should also collect them. WDYT?